packages/integrations/mdx/README.md

```diff
@@ -183,6 +183,42 @@ These are plugins that modify the output [estree](https://github.com/estree/estr
 
 We suggest [using AST Explorer](https://astexplorer.net/) to play with estree outputs, and trying [`estree-util-visit`](https://unifiedjs.com/explore/package/estree-util-visit/) for searching across JavaScript nodes.
 
+### `optimize`
+
+Whether to optimize the MDX output for faster builds and rendering. An internal rehype plugin will be injected last to transform static parts of the `hast` into [`set:html` properties](https://docs.astro.build/en/reference/directives-reference/#sethtml). As this modifies the intermediate `estree` and leaves some generated HTML to be un-escaped, the option is disabled by default.
```

Comment:
Good advice. I didn't notice the readme has the `**Type:**` pattern, I'll use that. The type is actually a `boolean | object`. I'll add a few more examples too.

I think there's still some useful information in the technical parts because it explains why the object option exists, but I'll try to move things around.